free-1.md

```diff
@@ -10,34 +10,45 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+I imagine the experience as follows: A user records themselves singing or playing an instrument. They then use the DSL in order to add thirds, fourths, and other intervals as they choose. They can also do other simpler things such as slowing down and speeding up 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think a DSL is appropriate because it would allow for lots of different types of behaviours from the user, and would also allow them lots of precision. For example, if a user wants an octave above a recorded sound at 50 percent volume with .1second reverb, that would be easy to do and straightforward to write. I think it addresses the need by making it really intuitive to do.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+The idea came straight out of the sound lab we did in class. As someone who loves singing and is especially obsessed with harmonies and chord progressions, this seemed like something I would love to use. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Modifying and complementing recorded sounds 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would start by recording and uploading sound files (in some format) and loading them in/putting them in the correct directory. The user would then use the commands written in to the language in order to modify, add to, and harmonize with the the sound file, by allowing the language to detect the frequency of the sounds in the file and apply transformations on it.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+As I wrote in the previous section, the frequency would be detected and tracked by the language after it is loaded in to the language. Then, once transformations are applied to the sound (by adding or reducing sounds at relative frequencies to the sound, by manipulating volume and reverb, and so on), the sound is then stored or played back (depending what the user wants). If multiple transformations on the sound are required, they are done in sequence.
+
+Errors would occur if, for example, a sound that doesn't exist is attempted, or if a user does not provide the correct inputs for modifying a sound (like, for example, not specifying a volume increase value when calling a increasVolume function). A standard error message should do here.
```

Comment:
Although a comment like this may be suited better for the end of this idea, I think it might be powerful to think of other ways you can interact with your language beyond a simple console interface. Since this DSL seems to be targeted more towards a general music audience than programmers themselves, why scare users off with spooky CS Error messages?